src/meta/src/hummock/hummock_manager.rs

```diff
@@ -178,7 +178,8 @@ macro_rules! start_measure_real_process_timer {
 #[derive(Default)]
 struct Versioning {
     current_version_id: CurrentHummockVersionId,
-    hummock_versions: BTreeMap<HummockVersionId, HummockVersion>,
+    current_version: HummockVersion,
+    hummock_versions: BTreeMap<HummockVersionId, Vec<u64>>,
```

Comment:
`Vec<u64>` -> `Vec<HummockSSTableId>` to make the code more readable. Can you also add a doc to explain:
1. what versions are kept in this map?
2. why we need to track SST ids for each version?

Reply:
> `Vec<u64>` -> `Vec<HummockSSTableId>` to make the code more readable. Can you also add a doc to explain:
> 
> 1. what versions are kept in this map?

all

> 2. why we need to track SST ids for each version?

given pin counts of each version, respectively, we need to determine whether a SST can be deleted in vacuum process

Reply:
> > `Vec<u64>` -> `Vec<HummockSSTableId>` to make the code more readable. Can you also add a doc to explain:
> > 
> > 1. what versions are kept in this map?
> 
> all
> 
> > 2. why we need to track SST ids for each version?
> 
> given pin counts of each version, respectively, we need to determine whether a SST can be deleted in vacuum process

 I know. I mean you can add documentation in the codes :)

Reply:
> we need to determine whether a SST can be deleted in vacuum process

fixed. just added some